src/controllers/controller.cpp

```diff
@@ -85,6 +85,7 @@ void Controller::send(QList<int> data, unsigned int length) {
     // If you change this implementation, also change it in HidController (That
     // function is required due to HID devices having report IDs)
 
+    length = data.size();
```

Comment:
aaaa. wait. or you could use the minimum of both, to keep the old functionality.